doc/administrator/web/ubuntu/index.md

```diff
@@ -6,25 +6,65 @@ description: "Ubuntu-specific install guide for setting up IRIDA."
 
 Ubuntu Installation Guide
 =========================
-Starting with a fresh, completely up-to-date system (Ubuntu Server 18.04).
+Starting with a fresh, completely up-to-date system (Ubuntu 24.04).
 
 Installing Software with `apt`
 ------------------------------
-You can install the latest OpenJDK JDK on Ubuntu. The `apt` repositories also contain an up-to-date version of Tomcat 8, so no complex configuration needs to take place for Ubuntu.
 
-    #### Install OpenJDK Java 11 
-    apt-get install openjdk-11-jdk
+    #### Install OpenJDK Java 17
+    sudo apt-get install openjdk-17-jdk
 
-    #### Install tomcat8 and mariadb-server
-    apt-get install --yes tomcat8 libtcnative-1 mariadb-server
+    #### Install mariadb-server
+    sudo apt-get install --yes mariadb-server
+
+    #### Download and Install tomcat9
+    sudo mkdir /opt/tomcat
+
+    For security reasons, it's best not to run Tomcat as the root user so create a dedicated user and group
+    sudo useradd -m -U -d /opt/tomcat -s /bin/false tomcat
+
+    Download [Tomcat 9](https://tomcat.apache.org/download-90.cgi)
+    sudo tar -xvf /PATH/TO/DOWNLOADED/apache-tomcat-9.0.98.tar.gz -C /opt/tomcat
+
+    Change ownership of Tomcat Directory
+    sudo groupadd tomcat
+    sudo usermod -a -G tomcat tomcat
+    sudo chown -R tomcat:tomcat /opt/tomcat
+
+    Update the ownership for the data directories that you have set in `/etc/irida/irida.conf`
+
+    Configure Tomcat as a service
+    sudo touch /etc/systemd/system/tomcat.service
+    nano /etc/systemd/system/tomcat.service

```

Comment:
**question:** Doesn't this need to be `sudo nano /etc/systemd/system/tomcat.service`?

Reply:
Updated in f4860e6e7b24b8fecd0a2984e22e8e4b76e07014